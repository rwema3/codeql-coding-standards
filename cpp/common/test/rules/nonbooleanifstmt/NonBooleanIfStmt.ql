// GENERATED FILE - DO NOT MODIFY
import codingstandards.cpp.rules.nonbooleanifstmt.NonBooleanIfStmt

class TestFileQuery extends NonBooleanIfStmtSharedQuery, TestQuery { }
