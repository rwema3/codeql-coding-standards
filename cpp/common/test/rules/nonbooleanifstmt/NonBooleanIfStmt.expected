| test.cpp:9:7:9:7 | i | If condition has non boolean type int. |
| test.cpp:11:7:11:7 | call to f | If condition has non boolean type int. |
| test.cpp:14:7:14:7 | a | If condition has non boolean type void *. |
